sunpy/net/helioviewer.py

```diff
@@ -127,16 +112,15 @@ def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
         >>> import sunpy.map
         >>> from sunpy.net import helioviewer
         >>> hv = helioviewer.HelioviewerClient()  # doctest: +REMOTE_DATA
-        >>> filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', instrument='AIA', detector='AIA', measurement='171')   # doctest: +REMOTE_DATA
```

Comment:
Can we add this back but using the correct sourceID now?

Reply:
I guess we can't add this because I cannot find the sourceId for it

Reply:
It has to be somewhere. I would have thought that to be a typical ask of Helioviewer. Might have ot email the people behind the project and ask.